tests/unit/test_nautobot_inventory.py

```diff
@@ -144,7 +144,7 @@ def test_nornir_nautobot_missing_token():
 def test_api_session(nornir_nautobot_class):
     expected_headers = {
         "User-Agent": f"python-requests/{requests.__version__}",
-        "Accept-Encoding": "gzip, deflate",
+        "Accept-Encoding": "gzip, deflate, zstd",
```

Comment:
@smk4664 why this change?

Reply:
That was added as part of https://github.com/nautobot/nornir-nautobot/pull/173